hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/Constants.java

```diff
@@ -99,17 +99,25 @@ private Constants() {
   public static final String MAX_PAGING_KEYS = "fs.s3a.paging.maximum";
   public static final int DEFAULT_MAX_PAGING_KEYS = 5000;
 
-  // the maximum number of threads to allow in the pool used by TransferManager
-  public static final String MAX_THREADS = "fs.s3a.threads.max";
-  public static final int DEFAULT_MAX_THREADS = 10;
+  // the maximum number of threads to allow in the pool used by TransferManager for uploads
+  public static final String UPLOAD_MAX_THREADS = "fs.s3a.threads.max";
+  public static final int UPLOAD_DEFAULT_MAX_THREADS = 10;
+
+  // the maximum number of threads to allow in the pool used by TransferManager for copies
+  public static final String COPY_MAX_THREADS = "fs.s3a.threads.max";
```

Comment:
Whoops, fixed. Leaving the parameters for the upload `TransferManager` the same since changing the name would be a backwards incompatible change.